arch/arm/soc/microchip_cec/Kconfig.soc

```diff
@@ -0,0 +1,9 @@
+config SOC_SERIES_CEC
+	bool "Microchip CECxxxx MCU"
+	select CPU_CORTEX_M
+	select CPU_CORTEX_M4
+	select CPU_FAMILY_CEC
```

Comment:
CPU_FAMILY_CEC seems undefined

Reply:
Right. This was little bit hazy for me on how and where to define `SOC_FAMILY`, `SOC_SERIES` and which type of `CPU_{FAMILY,SERIES}_CEC` things to define. Having that config split to `Kconfig`, `Kconfig.soc`, `Kconfig.defconfig`, etc. could be little bit less cluttered. I ended up not defining the family at all since there seems to be only two CEC CPUs at the moment. Maybe I'll just change this to `CPU_SERIES_CEC`